### PR TITLE
Create fumik0.py feed

### PR DESCRIPTION
**Pull request by doomedraven**

### CODE REVIEW

File: plugins/feeds/public/fumik0.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                         u"url": u"byhlavash.chimkent.su/vdvdv.exe"}}
+                """
+        if "http" not in block.get("server", {}).get("url"):
+            url = "http://" + block["server"]["url"]
+        context = {}
+        context["date_added"] = datetime.strptime(
+            block["first_seen"], "%Y-%m-%d %H:%M:%S")
+        context["as"] = block["server"]["AS"]
+        context["country"] = block["server"]["country"]
+        context["ip"] = block["server"]["ip"]
```
Comment (tomchop) [inline plugins/feeds/public/fumik0.py]: Are you 100% sure that these keys will exist? You seem to be checking for existence in L63

File: plugins/feeds/public/fumik0.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        context["source"] = self.name
+        context["md5"] = block["hash"]["md5"]
+        context["sha1"] = block["hash"]["sha1"]
+        context["sha256"] = block["hash"]["sha256"]
+
+        url_data = False
```
Comment (tomchop) [inline plugins/feeds/public/fumik0.py]:
```suggestion
        url_data = None
```

File: plugins/feeds/public/fumik0.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+             u"id": u"5c8915d47a324f51d460e8e5",
+             u"sample": {u"name": u"vdvdv.exe", u"size": u"600576"},
+             u"server": {u"AS": u"AS197695",
+                         u"country": u"ru",
+                         u"ip": u"37.140.192.146",
+                         u"url": u"byhlavash.chimkent.su/vdvdv.exe"}}
+                """
+        if "http" not in block.get("server", {}).get("url"):
+            url = "http://" + block["server"]["url"]
+        context = {}
```
[the remark this answers is not in the document]
Comment (doomedraven) [reply plugins/feeds/public/fumik0.py]: pushed small update

File: plugins/feeds/public/fumik0.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def analyze(self, block):
                          u"url": u"byhlavash.chimkent.su/vdvdv.exe"}}
                 """
         url = block["server"]["url"]
-        if "http" not in block["server"]["url"]):
+        if "http" not in block["server"]["url"]:
```
Comment (tomchop) [inline plugins/feeds/public/fumik0.py]: Why not use the `url` variable you created just before? (same for L44)

**Pull request merged**